CIP-0030/README.md

```diff
@@ -247,6 +245,30 @@ Errors: `APIError`, `TxSendError`
 
 As wallets should already have this ability, we allow dApps to request that a transaction be sent through it. If the wallet accepts the transaction and tries to send it, it shall return the transaction id for the dApp to track. The wallet is free to return the `TxSendError` with code `Refused` if they do not wish to send it, or `Failure` if there was an error in sending it (e.g. preliminary checks failed on signatures).
 
+
+
+## Events
```

Comment:
Should these events be exposed as an enum or constants as well? Similarly to how errors are